### PR TITLE
For #293: Implementing reservation Settings template screens

### PR DESCRIPTION
**Pull request by paulodamaso**
For #293: 
- Added template for `ReservationSettings` list screen
- Added template for `ReservationSetting` edit / create screen

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#452](https://github.com/timelesslounge/timelessis/issues/452) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#452](https://github.com/timelesslounge/timelessis/issues/452) is assigned to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil), here is [why](https://www.0crat.com/footprint/CF9LG1YCA/152bc4f8-2e3d-465b-9576-bec2e24edee1); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: timeless/reservations/views.py [hunk 1]
```diff
@@ -18,10 +18,6 @@
 class SettingsList(views.ListView):
     """
     List view set for Reservation Settings
-    @todo #186:30min For SettingsListView, SettingsCreateView,
-     SettingsDetailView, SettingsDeleteView create correct templates
```
Comment (amihaiemil) [inline timeless/reservations/views.py]: @paulodamaso what about the detail view and the delete view? Shouldn't you leave a puzzle for them?

**Comment by amihaiemil** [comment]
@paulodamaso also, there is a conflict

**Comment by paulodamaso** [comment]
@amihaiemil Done, please take a look

**Comment by amihaiemil** [comment]
@rultor good to merge

**Comment by emilianodellacasa** [comment]
@rultor merge

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @amihaiemil[/z](https://www.0crat.com/u/amihaiemil), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by ypshenychka** [comment]
@amihaiemil According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

Only one issue was found during code review. 
Please confirm that you'll try to find at least three major problems while future reviews.

**Comment by amihaiemil** [comment]
@ypshenychka I confirm

**Comment by ypshenychka** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Quality review completed: +6 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +15 point(s) just awarded to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil)

**Pull request merged**